code/__DEFINES/traits.dm

```diff
@@ -342,6 +342,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_FISH_CASE_COMPATIBILE "fish_case_compatibile" //Stuff that can go inside fish cases
 /// Plants that were mutated as a result of passive instability, not a mutation threshold.
 #define TRAIT_PLANT_WILDMUTATE "wildmutation"
+#define TRAIT_APC_SHOCKING "apc_shocking" //If you hit an APC with exposed internals with this item it will try to shock you
```

Comment:
This will make it a doc comment.

```suggestion
/// If you hit an APC with exposed internals with this item it will try to shock you
#define TRAIT_APC_SHOCKING "apc_shocking"
```